packages/integration-tests/src/ui-helpers/index.ts

```diff
@@ -27,14 +27,10 @@ type WaitToasterOptions = {
 
 export const waitForToaster = async (page: Page, { text, isError }: WaitToasterOptions) => {
```

Comment:
Would it be better to call it `expectToast`? Also, `isError` can be `type: 'success' | 'error' = 'success'`

Reply:
Updated.
Also rename `waitForToaster` to `waitForToast`.
Keep `waitFor` naming pattern since we want to align with the puppteer API.